packages/core/src/synchronizer.ts

```diff
@@ -187,7 +191,7 @@ export class UnirepSocialSynchronizer extends Synchronizer {
             },
             update: {
                 // add one for the current comment we're updating
-                commentCount: commentCount + 1,
+                commentCount: commentCount + count,
```

Comment:
We don't need a `count` variable in this function, it can be replaced with `findComment ? 0 : 1`